cmake/SDL.cmake

```diff
@@ -41,7 +42,11 @@ macro(sdl_gnu_common_ccxx_flags var)
             append(${var} "-fstack-protector-strong")
         endif()
     endif()
-    if(NOT (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 8.0))
+    # The -fcf-protection=full option fails to compile on aarch64 with the
+    # error: "'-fcf-protection=full' is not supported for this target".
+    # This patch skips this option for this configuration.
+    if(NOT (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 8.0 OR
+        CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64"))
```

Comment:
Per [GCC documentation](https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html) this option is x86 specific, so we should guard it accordingly.

+@spalicki

Reply:
@vpirogov It would be good to get this merged in ASAP, as this is breaking our builds. Thanks.

Reply:
@vpirogov @spalicki This intention is now reflected in the guard and the comment.